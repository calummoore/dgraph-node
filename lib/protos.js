'use strict';

Object.defineProperty(exports, "__esModule", {
  value: true
});

var _extends = Object.assign || function (target) { for (var i = 1; i < arguments.length; i++) { var source = arguments[i]; for (var key in source) { if (Object.prototype.hasOwnProperty.call(source, key)) { target[key] = source[key]; } } } return target; }; // @noflow


var _grpc = require('grpc');

var _grpc2 = _interopRequireDefault(_grpc);

var _path = require('path');

var _path2 = _interopRequireDefault(_path);

function _interopRequireDefault(obj) { return obj && obj.__esModule ? obj : { default: obj }; }

const p1 = _grpc2.default.load(_path2.default.join(__dirname, '/../protos/api.proto'));
exports.default = _extends({}, p1.api);