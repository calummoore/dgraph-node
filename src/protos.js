// @noflow
import grpc from 'grpc'
import path from 'path'

const p1 = grpc.load(path.join(__dirname, '/../protos/api.proto'))
export default { ...p1.api }
