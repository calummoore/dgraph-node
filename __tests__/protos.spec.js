import protos from '../src/protos'

describe('protos', () => {
  it('match snapshot', () => {
    expect(protos).toMatchSnapshot()
  })
})
