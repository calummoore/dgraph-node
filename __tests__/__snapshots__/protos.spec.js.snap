// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`protos match snapshot 1`] = `
Object {
  "Assigned": [Function],
  "AssignedIds": [Function],
  "Check": [Function],
  "ConnectionState": [Function],
  "Dgraph": [Function],
  "DirectedEdge": [Function],
  "ExportPayload": [Function],
  "Facet": [Function],
  "Facets": [Function],
  "FacetsList": [Function],
  "FilterTree": [Function],
  "Function": [Function],
  "Group": [Function],
  "GroupKeys": [Function],
  "KC": [Function],
  "KV": [Function],
  "KeyValues": [Function],
  "Latency": [Function],
  "LinRead": [Function],
  "List": [Function],
  "MapEntry": [Function],
  "Member": [Function],
  "MembershipState": [Function],
  "MovePredicatePayload": [Function],
  "Mutation": [Function],
  "Mutations": [Function],
  "NQuad": [Function],
  "Num": [Function],
  "Operation": [Function],
  "OracleDelta": [Function],
  "Order": [Function],
  "Param": [Function],
  "Payload": [Function],
  "Posting": [Function],
  "PostingList": [Function],
  "Proposal": [Function],
  "Query": [Function],
  "Raft": [Function],
  "RaftContext": [Function],
  "Request": [Function],
  "Response": [Function],
  "Result": [Function],
  "SchemaNode": [Function],
  "SchemaRequest": [Function],
  "SchemaResult": [Function],
  "SchemaUpdate": [Function],
  "SortMessage": [Function],
  "SortResult": [Function],
  "SrcFunction": [Function],
  "Tablet": [Function],
  "TaskValue": [Function],
  "TxnContext": [Function],
  "TxnTimestamps": [Function],
  "Value": [Function],
  "ValueList": [Function],
  "Version": [Function],
  "Worker": [Function],
  "Zero": [Function],
  "ZeroProposal": [Function],
}
`;
